src/map/mob_spell_container.cpp

```diff
@@ -67,7 +67,7 @@ void CMobSpellContainer::AddSpell(SpellID spellId)
         m_damageList.push_back(spellId);
 
     }
-    else if(spell->isNa() || spellId == SpellID::Erase){
+    else if(spell->isNa()){
```

Comment:
If you're removing Erase from the isNa() function, please add Erase check for Divine Veil here:
https://github.com/DarkstarProject/darkstar/blob/0807c177c8b4202d073c8ff54a79145d68c538fe/src/map/utils/battleutils.cpp#L4521

Though I'm not sure what the devs would prefer.

Reply:
isNa() checks for Erase as part of the function call. I didn’t actually remove anything except redundant code.

Reply:
isNa() checks for Erase as part of the function call. I didn’t actually remove anything except redundant code.

Reply:
Ooh, I missed that. My bad.